examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
hmm ... did you look at `make_projector_info`? Isn't that doing a `picks` on the projector? Also, you do have access to the channels in the projector object. See: raw.info['projs'][0]['data']['col_names'], which can be used to throw an error. It's late in India, will look at the thread tomorrow :)